### PR TITLE
Wizard Rebalance

### PR DESCRIPTION
**Pull request by Sparrowbird611**
## About The Pull Request

This PR is targeted primarily at the most commonly and least commonly used wizard abilities. Check the changelog for the full list. I didn't make any new sprites for the infinite air tank, so I just used the yellow double airtank for it.

## Why It's Good For The Game

Rebalances wizard spells to encourage spell variety, nerfing some of the more powerful and common ones while buffing the lesser used ones. Should encourage some very creative spell casting.

## Changelog
:cl:
add: The Wizard Federation has released the fifth edition of the Wizard's Handbook, altering the following items and spells.
add: Mastercraft wizard armor now comes with a breath mask and a nigh infinite oxygen tank, as well as a built in jetpack.
balance: spell card: Cooldown raised to 9 seconds, but price reduced to 1 magic point. Deals brute instead of burn now.
balance: Lightning bolt: Price decreased to 2 magic points, cooldown decreased to 15 seconds. Robeless.
balance: Fireball: Cooldown increased 18 seconds. Robed.
balance: Spacetime distortion: Cooldown increased to 45 seconds, now requires an invocation to cast (can't be cast while muted). Robeless.
balance: The traps: Robeless.
balance: Bind soul: Price increased to 3 magic points.
balance: Minimum cooldown adjusted for all spells listed.
/:cl:

### CODE REVIEW

**Comment by zeskorion** [comment]
Fireball was nerfed a bit too hard. You know we merged explosion nerfs right?

**Comment by hackerjsimon** [comment]
i like this yes

**Comment by Sparrowbird611** [comment]
Yeah, I know. I still think it's fine though, considering it's both a stun, and a high damage ability. Not to mention it burns clothes off.

**Comment by Sparrowbird611** [comment]
The only reason I left lightning bolt higher on the cooldown than fireball is because it's pretty much impossible to miss with a lightning bolt.

**Comment by Sparrowbird611** [comment]
I may consider lowering the price point of fireball back down to two, but we'll see. I think it would be better off as a three pointer, since it's basically hand grenade, good for both good CC and high damage.

**Comment by ATH1909** [comment]
> I may consider lowering the price point of fireball back down to two, but we'll see. I think it would be better off as a three pointer, since it's basically hand grenade, good for both good CC and high damage.

If you keep it as a three pointer, I'd recommend keeping it as a robeless spell, since that's kind of one of the biggest strengths of the fireball spell in SS13 (that it's one of the only direct offensive spells that you can cast while shapeshifted or while you're a golem).

> add: The wizard federation has **released** the fifth **edition** of the wizard's handbook, altering the following items and spells**:**

You might want to capitalize "Wizard's Handbook" as well; is that a title?

**Comment by Sparrowbird611** [comment]
> If you keep it as a three pointer, I'd recommend keeping it as a robeless spell, since that's kind of one of the biggest strengths of the fireball spell in SS13 (that it's one of the only direct offensive spells that you can cast while shapeshifted or while you're a golem).

Well, I don't plan on ever making it robeless again. I'd argue it's the strongest spell in the game, combining the second lowest cooldown for an offensive spell, (I nerfed that, a bit, but it's still fairly short) it's a stun, (thought it is soft instead of hard) and it usually only takes 1-2 shots to crit. I personally think it's fine as a three pointer, even in it's now weaker form.

> You might want to capitalize "Wizard's Handbook" as well; is that a title?

Yes, it's the title. I forgot to do that, so I'll fix that now, thank you for mentioning that and the other spelling errors I made.

**Comment by honkhonkclown** [comment]
not gonna touch the spell that literally dusts people?

**Comment by Sparrowbird611** [comment]
I didn't consider EI NATH a huge deal. Typically, you need to either combine it with another stun spell, or just wait for a person to be alone to use it, since you risk getting mobbed. As one of only two melee ranged spells that rarely get picked, I didn't see that big of a deal with it, given you'll only ever kill one person with it before waiting for the recharge.

**Comment by Sparrowbird611** [comment]
After a few fairly convincing arguments, I decided to revert the price of fireball. However, I increased it's cooldown to eighteen seconds as compensation.

File: code/modules/projectiles/projectile/magic/spellcard.dm [hunk 1]
```diff
@@ -2,5 +2,5 @@
 	name = "enchanted card"
 	desc = "A piece of paper enchanted to give it extreme durability and stiffness, along with a very hot burn to anyone unfortunate enough to get hit by a charged one."
 	icon_state = "spellcard"
-	damage_type = BURN
+	damage_type = BRUTE
```
Comment (ike709) [inline code/modules/projectiles/projectile/magic/spellcard.dm:5]: You switched the damage type but the desc still says it burns.

Comment (Sparrowbird611) [reply code/modules/projectiles/projectile/magic/spellcard.dm:5]: I thought I fixed that description. I'll do that now.

Comment (Sparrowbird611) [reply code/modules/projectiles/projectile/magic/spellcard.dm:5]: Ok so, turns out, I fixed the spell description, but not the projectile description. It's good now.

**Pull request merged**